addons/website/static/src/core/website_youtube_video_service.js

```diff
@@ -0,0 +1,52 @@
+import { registry } from "@web/core/registry";
+import { loadJS } from "@web/core/assets";
+import { hasTouch } from "@web/core/browser/feature_detection";
+import { SIZES, utils as uiUtils } from "@web/core/ui/ui_service";
+
+registry.category("services").add("youtube_video", {
```

Comment:
prefix the service name with website: `website.youtube_video`. we should do that for every website service